macosx/HBFilters+UIAdditions.m

```diff
@@ -275,6 +316,44 @@ + (NSDictionary *)denoiseTypesDict
     return denoiseTypesDict;
 }
 
++ (NSDictionary *)sharpenPresetDict
+{
+    if (!sharpenPresetDict)
+    {
+        sharpenPresetDict = filterParamsToNamesDict(hb_filter_param_get_presets, HB_FILTER_NLMEANS);
```

Comment:
s/NLMEANS/UNSHARP/

I'm not sure how to switch between Unsharp and Lapsharp here.